packages/slate-sync/index.js

```diff
@@ -78,9 +69,9 @@ function _generateIgnoreFlags() {
  * @return          Promise
  */
 async function deploy(cmd = '', files = []) {
-  if (!['upload', 'replace'].includes(cmd)) {
+  if (!['deploy', 'replace'].includes(cmd)) {
```

Comment:
It looks like `replace` doesn't exist anymore? And has been replaced by `deploy`. And that `upload` has been replaced by `deploy --nodelete`?

Reply:
Yes, you're right. I'll update the the commands and documentation to reflect this when I get a chance